rts/lra/client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -1 +1,2 @@
-org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
+org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder
+org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
```

Comment:
Where do we use the dependency on MpClientBuilderImpl?

Reply:
It is the implementation of the ClientBuilder, so (for example) we use it [here](https://github.com/jbosstm/narayana/blob/30a7d8e676606d69017ea84d75b0218b1d163c9d/rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java#L886)

Reply:
The first implementation dependency `public class ResteasyClientBuilder extends javax.ws.rs.client.ClientBuilder` 
and `public class MpClientBuilderImpl extends ResteasyClientBuilderImpl` so why do we need the both? I'll ask again, how come it worked before: which use case requires MpClientBuilderImpl and which use cases require the super class ResteasyClientBuilderImpl. I assume that the TCK requires MpClientBuilderImpl (or is it the MP platform that requires it). So if that is the case then we just need to identify the ResteasyClientBuilderImpl use case.

We used to have just `ResteasyClientBuilder` but then someone on the team replaced it with just `MpClientBuilderImpl`. So if you're not sure about the use cases then can you look back through the commit history and tag the engineer who made the change and ask them if we need tell the service loader about both implementation classes.

Reply:
Of course, I can do that :-). Just to mention, it is possible to specify a list of dependencies separated with newline, therefore, specifying two dependencies for `javax.ws.rs.client.ClientBuilder` should not be a problem

Reply:
My concern is that depending on MpClientBuilderImpl is coupling the `lra-client` module to whatever library it belongs to. Your change back to the generic case makes the inclusion of MpClientBuilderImpl suspicious.

Reply:
So your analysis shows MpClientBuilderImpl was added whilst fixing https://issues.redhat.com/browse/JBTM-3381 and was required for use with quarkus (thanks for researching that). So it looks like your proposal (to add both) is the correct one. I will mark this thread as resolved.